recipe/meta.yaml

```diff
@@ -1,53 +1,64 @@
 {% set name = "lcms2" %}
-{% set version = "2.12" %}
+{% set version = "2.16" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://github.com/mm2/Little-CMS/archive/{{ version }}.tar.gz
-  sha256: e501f1482fc424550ef3abbf86bf1c66090e1661249e89552d39ed5bf935df66
+  url: https://github.com/mm2/Little-CMS/releases/download/lcms{{ version }}/lcms2-{{ version }}.tar.gz
+  sha256: d873d34ad8b9b4cea010631f1a6228d2087475e4dc5e763eb81acc23d9d45a51
   patches:
     - 0001_windows_linux_compat.patch          # [win]
     - 0002_change_windows_sdk_version.patch    # [win]
 
 build:
   number: 0
   skip: true  # [win and vc<14]
-  skip: True  # [win32]
   run_exports:
     - {{ pin_subpackage("lcms2") }}
 
 requirements:
   build:
     - {{ compiler('c') }}
-    - make  # [not win]
+    - make     # [not win]
     - libtool  # [unix]
   host:
-    - jpeg
-    - libtiff
+    - jpeg {{ jpeg }}
+    - libtiff 4.5.1
```

Comment:
```suggestion
    - libtiff {{ libtiff }}
```

Reply:
I rewrote to:
```
    # libtiff 4.5.1 fixes a missing libtiff.lib on Windows
    - libtiff 4.5.1          # [win]
    - libtiff {{ libtiff }}  # [not win]
```

Reply:
I think it has a cascading effect on all libtiff's downstreams on win-64: lcms2, poppler, pillow, gdal, imagecodecs

Reply:
Yes, I'll do

Reply:
Here we go https://github.com/AnacondaRecipes/aggregate/pull/324